### PR TITLE
TaskN4

### PR DESCRIPTION
**Pull request by AvetSargsyan**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+"use strict";
```
Comment (artiombond) [inline script.js:1]: uncomment all the code

File: script.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+let numArr = [1,6,8,0,-2].map(num =>Boolean(num));
+console.log(numArr);
+*/
+
+/*
+//5.Sort the array [1,6,7,8,3,4,5,6] in descending order
+let array =[1,6,7,8,3,4,5,6].sort((a,b)=> {
+    if (a<b) return 1;
+    if (a==b) return 0;
+    if (a>b) return -1;
```
Comment (artiombond) [inline script.js:37]: there is no need to write such conditions, you can write  sort((a,b)=> b - a);
you can read more on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

File: script.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+//6.Filter array [1,6,7,8,3,4,5,6] by value > 3
+let arr = [1,6,7,8,3,4,5,6].filter((num)=>{
+    if (num > 3) return true
+});
+console.log(arr);
+*/
+
+/*
+//7.Write a function that takes two parameters - an array and a number and outputs the index of an array element equal to a number
+//If I understand you correctly..
```
Comment (artiombond) [inline script.js:52]: for exmple if you have an array [0, 1, 2, 3, 5], and you pass 5 in your function, it should return index of element that equal 5, so it should be 4

File: script.js (hunk 2 above)
Comment (AvetSargsyan) [reply script.js:37]: OK, thank you.I’ll read it and fix it

**Pull request closed without merging**